### PR TITLE
Ui farm

### PR DESCRIPTION
**Pull request by seeden**
updated farm and plot pages
refactored many other components
prepare for monorepo (added core and icons folders)
removed old pages
updated deps

### CODE REVIEW

**Review by hoffmang9 — CHANGES_REQUESTED** [review]
Two things so far:
I blew away my .chia dir and my venv and installed from scratch. I get past the swirling "connecting to wallet" but then end up with a blank white page. The error in the console is `middleware.js:80 WebSocket connection to 'ws://localhost:55400/' failed: Error in connection establishment: net::ERR_CONNECTION_REFUSED`
Second - there are some LGTM.com issues to address

File: electron-react/src/components/farm/card/FarmCardCoinbaseRewards.tsx [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+  const { loading, value } = useAsync(() => computeStatistics(wallets), [
+    wallets,
+  ]);
+
+  const coinbaseRewards = useMemo((): number => mojo_to_chia(value?.coinbaseRewards), [value?.coinbaseRewards]);
+
+  return (
+    <FarmCard
+      title={
+        <Trans id="FarmCardCoinbaseRewards.title">XCH Framing Rewards</Trans>
```
Comment (hoffmang9) [inline electron-react/src/components/farm/card/FarmCardCoinbaseRewards.tsx:21]: We need to remain TXCH unless showing mainnet balances

**Comment by hoffmang9** [comment]
There seems to be a discrepancy between the netspace reported on the Node page and that reported on the farm page. Not sure which one is "right" but I do know that we need them to match.

File: electron-react/src/components/farm/card/FarmCardCoinbaseRewards.tsx (hunk 1 above)
Comment (seeden) [reply electron-react/src/components/farm/card/FarmCardCoinbaseRewards.tsx:21]: I already changed it

**Review by hoffmang9 — APPROVED** [review]
Looks good but needs merge conflicts resolved

**Pull request merged**